badges/badges1.css

```diff
@@ -0,0 +1,32 @@
+:root{
+    --primaryBadgeCol:#876445;
+    --secondaryBadgeCol:#D96098;
+    --thirdBadgeCol:#24A19C;
+    --fourthBadgeCol:#325288;
+}
+
+.badges{
+    color:rgb(202, 186, 186);
```

Comment:
Use variable for color here. Also use consistent units in colors: either all RGB or all hex values (hex is easier).